include/tvm/runtime/module.h

```diff
@@ -221,10 +221,20 @@ class TVM_DLL ModuleNode : public Object {
    * By default, none of the property is set. Derived class can override this function and set its
    * own property.
    */
-  virtual int GetProperty() const { return 0b000; }
+  virtual int GetPropertyMask() const { return 0b000; }
+
+  /*! \brief Returns true if this module is 'binary serializable'. */
+  bool IsBinarySerializable() const {
```

Comment:
I feel we don;t need this method. Instead, we can just expose GetProperty and let python side use this function